database/schema.json

```diff
@@ -0,0 +1,86 @@
+{
+    "collections": [
+      {
+        "name": "Activities",
+        "fields": [
+          { "name": "activityId", "type": "Document ID" },
+          { "name": "description", "type": "String" },
+          { "name": "tickets", "type": "Number" }
+        ]
+      },
+      {
+        "name": "Prizes",
+        "fields": [
+          { "name": "prizeId", "type": "Document ID" },
+          { "name": "name", "type": "String" },
+          { "name": "ticketCost", "type": "Number" }
+        ]
+      },
+      {
+        "name": "Users",
+        "fields": [
+          { "name": "userId", "type": "Document ID" },
+          { "name": "email", "type": "String" },
+          { "name": "displayName", "type": "String" },
+          { "name": "totalTickets", "type": "Number" }
+        ],
+        "subcollections": [
+          {
+            "name": "UserActivities",
+            "fields": [
+              { "name": "activityId", "type": "Document ID" },
+              { "name": "completedAt", "type": "Timestamp" },
+              { "name": "photos", "type": "Array of Strings - URLs" }
+            ]
+          },
+          {
+            "name": "UserEntries",
+            "fields": [
+              { "name": "entryId", "type": "Document ID" },
+              { "name": "prizeId", "type": "String" },
+              { "name": "ticketsUsed", "type": "Number" }
```

Comment:
should we also keep track of number of entries? is this something that we'll want to show in the UI?